### PR TITLE
Global scope leakage

### PR DESCRIPTION
**Pull request by gilmoreorless**
Hi there,

I noticed a couple of variables that weren't declared and were leaking into the global scope, and made a quick fix.

Cheers,

Gil

### CODE REVIEW

File: jquery.transform.js [hunk 1]
```diff
@@ -433,7 +434,7 @@ function unmatrix(matrix) {
 
 	// matrix is singular and cannot be interpolated
 	} else {
-		rotate = scaleX = scaleY = skew = 0;
+		scaleX = scaleY = skew = 0;
```
Comment (louisremi) [inline jquery.transform.js]: please replace this rotate var by
B = A = ...

**Comment by louisremi** [comment]
Thanks, will pull as soon as you amend your PR.
FYI, those vars aren't exactly leaked in the global scope, since the whole plugin is wrapped into a self-executing function.
But not defining a var is a bug, you're right.

**Comment by gilmoreorless** [comment]
OK, I've updated that rotate line.
And I respectfully disagree about the leakage - if that part of the code is run (in an animation for rotate, or in setting the matrix filter for IE with the centerOrigin variable) then the variable gets defined on the window object.

**Comment by louisremi** [comment]
Wow, you're right...
For some reason I always thought SEFs were protecting the global scope.
Learned something new today.

**Comment by louisremi** [comment]
Thanks!

**Pull request merged**